gulp/javascript.js

```diff
@@ -1,6 +1,6 @@
 const path = require('path')
 const webpack = require('webpack')
-const util = require('gulp-util')
+const argv = require('minimist')
```

Comment:
This is probably missing the `(process.argv.slice(2))` part

Reply:
@backflip nice catch, pushed the fix